src/engine-scripts/puppet/loginReload.js

```diff
@@ -0,0 +1,11 @@
+const loginReload = async (page, scenario) => {
+    const INPUT_SELECTOR = 'input[name="wpName"]';
+    await page.goto(scenario.loginUrl);
+    await page.waitForSelector(INPUT_SELECTOR);
+    await page.type(INPUT_SELECTOR, scenario.username);
+    await page.type('input[name="wpPassword"]', scenario.password);
+    await page.click('button[type="submit"]');
```

Comment:
```suggestion
    page.click('button[type="submit"]');
```

From debugging the timeouts I've been getting, it looks like having this `await` might be problematic. There might be a race condition happening where by the time this promise has resolved, the navigation is in the middle or at the end leading to L8 timing out. When removing this await, I haven't received a timeout yet, but I'm still checking

Reply:
Or actually maybe using something like as described at https://github.com/puppeteer/puppeteer/issues/1412#issuecomment-345357063

```
await Promise.all([
  page.waitForNavigation(),
  page.click('a')
]);
```